include/seqan3/range/decorator/gap_decorator.hpp

```diff
@@ -574,8 +727,35 @@ class gap_decorator
         return const_iterator{*this};
     }
 
-    /*!\brief Returns an iterator to the element following the last element of the decorator.
-     * \returns Iterator to the behind last element.
+    /*!\brief Returns a pseudo random access iterator to the first element of the container.
+     * \returns Iterator to the first element.
+     *
+     * If the container is empty, the returned iterator will be equal to end().
+     *
+     * \attention The returned iterator models the std::ranges::RandomAccessIterator but
+     *            **the operations on it do not guarantee constant time**.
```

Comment:
but what do they guarantee?

Reply:
They operate in `log(g)` where `g` is the number of gaps. I added this in the description